staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go

```diff
@@ -264,6 +270,15 @@ const (
 	// NamesAccepted means the names chosen for this CustomResourceDefinition do not conflict with others in
 	// the group and are therefore accepted.
 	NamesAccepted CustomResourceDefinitionConditionType = "NamesAccepted"
+	// NonStructuralSchema means that the CustomResourceDefinition has one or more schemas which are not structural.
```

Comment:
Is there anything we can point readers of this comment to that would help them understand what the criteria for structural is?

Reply:
We have the KEP. Later we will have the documentation. Do we cross-reference API docs with other docs usually?

Reply:
Good question. I see there is a modest amount of "More info: https://kubernetes.io/docs/.." occurrences in our code, but maybe this will be sufficiently easy to find via a web search later anyway. Might not be anything we can do now.

Reply:
Also note that I extended the text here now, at least to have a short variant of the definition. The docs will have all the details.